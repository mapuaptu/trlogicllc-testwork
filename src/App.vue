<template>
  <div id="app">
    <div class="container">
      <div class="headline">
        Todo builder
      </div>
      <router-view />
    </div>
  </div>
</template>
