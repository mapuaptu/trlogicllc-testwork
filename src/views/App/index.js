import Footer from '@/views/Footer/index.vue';

export default {
  name: 'App',
  components: {
    Footer,
  },
};
