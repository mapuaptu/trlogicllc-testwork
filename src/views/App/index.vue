<template>
  <div id="app">
    <modals-container />

    <div class="container">
      <div class="top">
        <div class="headline">
          <router-link :to="{name: 'home'}">
            Note builder
          </router-link>
        </div>
        <router-view />
      </div>

      <div class="bottom">
        <Footer github-link="https://github.com/" />
      </div>
    </div>
  </div>
</template>

<script src="./index.js"></script>
<style src="./index.scss" lang="scss" module></style>
