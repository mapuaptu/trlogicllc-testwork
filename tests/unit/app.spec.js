import App from '@/App.vue';
import { shallowMount } from '@vue/test-utils';

describe('App component', () => {
  it('1. Rendered success', async () => {
    const wrapper = await shallowMount(App, {
      stubs: ['router-view', 'modals-container'],
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
