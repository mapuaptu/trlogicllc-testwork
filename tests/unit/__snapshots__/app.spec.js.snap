// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component 1. Rendered success 1`] = `
<div id="app">
  <modals-container-stub></modals-container-stub>
  <div class="container">
    <div class="headline">
      Todo builder
    </div>
    <router-view-stub></router-view-stub>
  </div>
</div>
`;
